src/Categories/Categories.tsx

```diff
@@ -4,13 +4,14 @@ import './Categories.scss'
 
 interface CategoryProps {
   getCategory: (text: string) => void;
+  imageId: string
 }
 
-const Categories: React.FC<CategoryProps> = ({getCategory}) => {
+const Categories: React.FC<CategoryProps> = ({getCategory, imageId}) => {
   const categories = ['photography', 'watercolor', 'painting', 'oil painting', 'sculpture', 'pencil', 'pastels', 'chalk']
   const allCategories = categories.map(category => {
     return (
-      <CategoryCard key={category} category={category} acquireCategory={getCategory} /> //we need to pass down the function to here as a prop
+      <CategoryCard key={category} category={category} acquireCategory={getCategory} artImageId={imageId} /> //we need to pass down the function to here as a prop
```

Comment:
We can definitely remove the comment I left there, I don't know what we need it anymore.